.claude/shared/superclaude-rules.yml

```
@@ -0,0 +1,158 @@
+## Smart_Defaults
+File_Discovery: "Recent edits first|Common locations|Git status integration|Project patterns recognition"
+Command_Intelligence: "'test'→package.json scripts|'build'→project configuration|'start'→main entry point"
+Context_Intelligence: "Recent mentions|Error messages|Modified files|Project type detection"
+Interruption_Handling: "'stop'|'wait'|'pause'→Immediate acknowledgment|State preservation|Clean partial operations"
+Solution_Escalation: "Simple→Moderate→Complex progression|Try obvious approaches first|Escalation"
+Integration: "18-command system|MCP orchestration|Persona specialization|Evidence-based decisions"
+
+## Ambiguity_Resolution
+Detection: "'something like'|'maybe'|'fix it'|'etc' keywords|Missing: paths|scope|criteria"
+Resolution: "Options: 'A)[interpretation] B)[alternative] Which?'|Refinement: Broad→Category→Specific→Confirm"
+Context_Intelligence: "Recent operations|Files accessed → 'You mean [X]?'|Common patterns: 'Fix bug'→Which?|'Better'→How?"
```

YAMLlint (1.37.1)

[error] duplication of key "Context_Intelligence" in mapping

(key-duplicates)